src/sentry/static/sentry/app/actionCreators/group.jsx

```diff
@@ -1,5 +1,4 @@
-import Raven from 'raven-js';
-
+import * as sdk from 'app/utils/sdk';
```

Comment:
This is minor and I don't really care that much but it'd be more consistent if we exported a single default object instead of having to `import *`